### ISSUE TITLE
Defined type symbols can't be assigned to a typedesc

### ISSUE THREAD

**Issue by manuranga** [ballerina-platform/ballerina-lang#24095]
```ballerina
type intOrString int|string;

function testTupleUnionTypes() {
    typedesc<int|string> b = intOrString;
}
```
fails with
```
ERROR: .::typedesc_positive.bal:84:30:: undefined symbol 'intOrString'
```

please uncomment above test case in `ballerina-lang/tests/jballerina-unit-test/src/test/resources/test-src/types/typedesc/typedesc_positive.bal` after fixing this.

**Comment by pubudu91** [comment]
This issue was fixed with PR #24093. The above mentioned test case is not commented though. Can this issue be closed?

**Comment by manuranga** [comment]
yes fixed with that pr, thanks
